client/cypress/integration/login.spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+context('Login', () => {
+  beforeEach(() => {
+    cy.log("Hitting browser");
+    cy.visit('http://localhost:3000')
+  });
+
+  it("should join a game", () => {
+    cy.log("creating new game");
+    cy.get("#newGame").click();
+
+    cy.log("Confirm the new game is created");
+    cy.get("#status").contains("New game id is:");
+    cy.get("#gameId").should("have.value", "12234");
```

Comment:
It is not clear to me why the `gameID` would always be `12234`

Reply:
so. this is one I was waiting for someone to point out. I'm currently working on a more dymanic process to strip out the game id. :) Locally, my machine always just serves up that same value. I'll be adjusting this for the future.